extensions/GOOGLE/SPV_GOOGLE_hlsl_functionality1.asciidoc

```diff
@@ -41,7 +41,7 @@ Version 1.2 Revision 4, Unified.
 
 This extension requires SPIR-V 1.0.
 
-This extension also requires the SPV_GOOGLE_decorate_string extension.
+This extension also subsumes the SPV_GOOGLE_decorate_string extension.
```

Comment:
Not sure whether this is the correct place to put these words. Please let me know if it is not.